src/fheroes2/dialog/dialog_selectscenario.cpp

```diff
@@ -91,14 +91,31 @@ namespace
         fheroes2::showStandardTextMessage( header, body, Dialog::ZERO );
     }
 
-    void mapInfo( const Maps::FileInfo & info )
+    void PlayersToolTip( [[maybe_unused]] const Maps::FileInfo * info = nullptr )
```

Comment:
The `[[maybe_unused]]` attribute as useful when variable or parameter may be used _sometimes_ - e.g. in the debug builds only. For cases where the parameter is _never_ used, it is possible to use a more explicit indication of this - just do not name it:

```suggestion
    void PlayersToolTip( const Maps::FileInfo * /* info */ = nullptr )
```

The same applies to the rest of the cases below.

Reply:
Applied suggested changes here and for remaining cases